01-ipm.Rmd

```diff
@@ -0,0 +1,95 @@
+---
+title: "IPM"
+---
+
+# IPM {#ipm}
+
+An IPM, or Integrated Population Model, combines data from many sources in its analyses for the sake of  estimates that acknowledge the many factors that can influence abundance and survival. Our IPM uses data from survival and sightability models, as well as harvest data from the IDFG database to calculate rates like survival and sex ratios for an entire [<b>DAU</b>]{style="color: #086e04;"}, extrapolated to the entire unit and better informed by all the different metrics that are considered in the process of fitting the model.
+
+## Walkthrough {#ipm-model}
+
+Using the IPM tool can be split into two major processes: preparing data and looking at data.
+
+### Preparing Data {#ipm-load}
+
+Click on the Setup tab, the first dropdown under the IPM tool menu.
+
+<center>![](./www/ipm_walk1.png)</center>
+
+On the top left of the page should be a window labeled Overview, and this is where you determine the data that you run through the IPM. Select the database you want to retrieve the data from, then make sure you choose your Species, [<b>DAU</b>]{style="color: #086e04;"}, Unit, and Weather before you click [<b>Fetch Harvest Data</b>]{style="color: #196eb0;"} on the bottom right of the Overview window to retrieve the specific data you requested.
+
+<center>![](./www/ipm_walk2.png)</center>
+
+Once the data is loaded, you should get a popup dialog letting you know that the process was successful. It will also prompt you to proceed to the "Structure" tab (At the top left of the page next to the Overview tab), which is where we will continue our data preparation.
+
+The Structure tab allows you to select whether your survival and reproduction will have a constant or time varying data variance. The defaults were chosen to be well-suited to a variety of situations, so most of the time clicking [<b>"Default Structure"</b>]{style="color: #196eb0;"} will be all you need to do here. Often the inputs that appear when the tool is opened are actually not the same as those that are selected when you press [<b>"Default Structure"</b>]{style="color: #196eb0;"}, so make sure you click that before you move on to Harvest if those are the inputs you are planning to use.
+
+<center>![](./www/ipm_walk4.png)</center>
+
+After you select your Structure, you can move on to the Harvest Tab. Decide your effort variable and the years that you will be conducting your analysis, then look at the given information for the years that will be forecast. You can change the Harvest by double clicking the numbers and typing in new entries.
+
+<center>![](./www/ipm_walk5.png)</center>
+
+Our next step is the Run tab, which again already has a decent set of default values loaded. Adjust your Burnin Length, MCMC iterations, and Thinning Length. Decide whether you would like to automate convergence (This may increase the time it takes to run the model).
```

Comment:
It might be useful to define the MCMC terms (Burnin, iterations, thinning) in the Glossary.